deploy/crds/pravega_v1alpha1_pravegacluster_cr.yaml

```diff
@@ -47,21 +44,28 @@ spec:
     autoRecovery: true
 
   pravega:
-    controllerReplicas: 1
-    segmentStoreReplicas: 3
-
-    cacheVolumeClaimTemplate:
-      accessModes: [ "ReadWriteOnce" ]
-      storageClassName: "standard"
-      resources:
-        requests:
-          storage: 20Gi
-
     image:
       repository: pravega/pravega
       tag: latest
```

Comment:
Does this setting still work? 
https://github.com/pravega/pravega-operator/blob/issue-245-external-access/pkg/apis/pravega/v1alpha1/pravegacluster_types.go#L189

Reply:
`tag` can be specified but internally gets ignored.